setting/server_setting.txt

```diff
@@ -0,0 +1 @@
+bluetooth_available 1
```

Comment:
Refer to #42, this file put in bluetooth will better than in setting.
And the "bluetooth_enable" is better than "bluetooth_available"